src/Microsoft.AspNetCore.Mvc.TagHelpers/CacheTagHelper.cs

```diff
@@ -119,8 +119,8 @@ public CacheTagHelper(CacheTagHelperMemoryCacheFactory factory, HtmlEncoder html
                 // Use the CreateEntry to ensure a cache scope is created that will copy expiration tokens from
                 // cache entries created from the GetChildContentAsync call to the current entry.
                 IHtmlContent content;
-                using (var entry = MemoryCache.CreateEntry(cacheKey))
-                {
+                var entry = MemoryCache.CreateEntry(cacheKey);
```

Comment:
Can you add a comment that says why we aren't using a `using` here. Just so that nobody changes it in the future.